docs/docs/References/Plugin_Operations.md

```diff
@@ -7,7 +7,9 @@
 !!! warning
     For each operation, the argument names must match exactly. For example, the Repository Discovery
     operation must have a single argument named `source_connection`.
-
+    
+!!! warning
```

Comment:
This is more applicable to the `Schemas` I think. The plugin operations themselves do not control the amount of input/output